VideoWeb/VideoWeb.Common/Models/CustomClaim.cs

```diff
@@ -0,0 +1,17 @@
+namespace VideoWeb.Common.Models;
+
+/// <summary>
+/// This class is to represent a parameterless constructor for the Claim class.
+/// </summary>
+public class CustomClaim
```

Comment:
Added this to resolve:
`Newtonsoft.Json.JsonSerializationException: Unable to find a constructor to use for type System.Security.Claims.Claim. A class should either have a default constructor, one constructor with arguments or a constructor marked with the JsonConstructor attribute`.

Suspected we couldn't deserialise and so called the API again anyway